backend/src/main/java/com/google/blackswan/servlets/CronServlet.java

```diff
@@ -40,17 +44,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     log.info("Cron job ran.");
     
     // TODO: Logic for cron job to run blackswan mock. (#13)
-    storeAlertInDatastore();
-    
+
+    // Simple logic for cron job, since we only have one set of alerts for now. 
+    clearCurrentAlertsInDatastore();
+    storeAlertsInDatastoreSimple();
+    fetchAlertsFromDatastore();
     response.setStatus(HttpServletResponse.SC_ACCEPTED); 
   }
 
-  /** Temporary method for storing dummy alert into datastore. */
-  private void storeAlertInDatastore() {
-    AlertGenerator testAlertGenerator = new DummyAlertGenerator(new DummyAnomalyGenerator());
-    List<Alert> alerts = testAlertGenerator.getAlerts();
+  private void clearCurrentAlertsInDatastore() {
+    Query query = new Query(Alert.ALERT_ENTITY_KIND);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    // Iterate through the entities to delete all comment objects.
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      Key alertEntityKey = KeyFactory.createKey(Alert.ALERT_ENTITY_KIND, id);
+      datastore.delete(alertEntityKey);
+    }
+
+  }
+
+  /** Temporary method for storing alerts from simpleGenerator into the datastore. */
+  private void storeAlertsInDatastoreSimple() {
+    AlertGenerator simpleGenerator = new SimpleAlertGenerator(new SimpleAnomalyGenerator());
+    simpleGenerator.getAlerts().forEach(alert -> {
+      DatastoreServiceFactory.getDatastoreService().put(alert.toEntity());
+    });
+  }
+
+  /** Sample fetch alerts from datastore and creating alert objects from it. */
+  private void fetchAlertsFromDatastore() {
+    Query query = new Query(Alert.ALERT_ENTITY_KIND);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    List<Alert> alertList = new ArrayList<>();
+
+    for (Entity entity : results.asIterable()) {
+      alertList.add(Alert.createAlertFromEntity(entity));
+    }
 
-    DatastoreServiceFactory.getDatastoreService().put(alerts.get(0).toEntity());
+    // Print out to see if alerts are correctly converted.
+    alertList.forEach(alert -> System.out.println(alert));
```

Comment:
was this for debugging purposes? should this function return the alertList instead...?

Reply:
this function and print statement was mainly for the demo, and like a `sample fetching function` that other function can reference lol...
i will get rid of the print statements, and add some comments to clarify!